src/main/java/core/Alphabetizer.java

```diff
@@ -0,0 +1,21 @@
+package core;
+
+public class Alphabetizer {
+
+    private static final CustomCharacterComparator CUSTOM_CHARACTER_COMPARATOR = new CustomCharacterComparator();
+
+    public static String alphabetize(String input){
+        if(input.length()<2){
+            return input;
+        }
+
+        return input.chars()
```

Comment:
It was unclear whether stream implementation is indeed faster since a simple google search online would show otherwise. However, [jmh performance tests show otherwise](https://github.com/akohli96/Alphabetizer/pull/3/files#diff-cda9ecd6b8b20a41aa19a893062db5d5)